app/profile/components/profile.js

```diff
@@ -0,0 +1,29 @@
+const h = require('vdux/element').default
+
+const Highlight = require('../../highlights/components/Highlight')
```

Comment:
unless "within a module" (such as `profile`), always do "top-down" `require`s (as in `require('app/highlights/components/Highlight')`